### PR TITLE
Ar 1032 source picture view toolbar in heb eng for prod

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/components/Pages/WithText/Buttons/ToggleScan/AddScanBookmarkBtn.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+const AddScanBookmarkBtn = () => {
+  const { t } = useTranslation();
+  const dir   = useSelector(state => settings.getUIDir(state.settings));
+
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => setOpen(true);
+
+  const handleClose = (e, el, isCreated) => {
+    if (!isCreated) {
```
Comment (gshilin) [inline src/components/Pages/WithText/Buttons/ToggleScan/AddScanBookmarkBtn.js:20]: Why do you need this `if`? Both inside it and outside you're calling `setOpen(false)`

File: src/components/Pages/WithText/Buttons/ToggleScan/ScanPDF.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      return null;
+    }
+
+    const setDivSize = () => setWidth(ref.current.getBoundingClientRect().width);
+    setDivSize();
+    window.addEventListener('resize', setDivSize);
+    return () => {
+      window.removeEventListener('resize', setDivSize);
+    };
+  }, [file]);
```
Comment (gshilin) [inline src/components/Pages/WithText/Buttons/ToggleScan/ScanPDF.js:38]: How does this useEffect depends on `file`?

Comment (davgur) [reply src/components/Pages/WithText/Buttons/ToggleScan/ScanPDF.js:38]: fixed

File: src/components/Pages/WithText/Buttons/ToggleScan/AddScanBookmarkBtn.js (hunk 1 above)
Comment (davgur) [reply src/components/Pages/WithText/Buttons/ToggleScan/AddScanBookmarkBtn.js:20]: fixed

**Review by gshilin — APPROVED** [review]

**Pull request merged**